alibi_detect/cd/sklearn/classifier.py

```diff
@@ -232,3 +266,19 @@ def score(self, x: Union[np.ndarray, list]) -> Tuple[float, float, np.ndarray, n
         p_val, dist = self.test_probs(y_oof, probs_oof, n_ref, n_cur)
         probs_sort = probs_oof[np.argsort(idx_oof)]
         return p_val, dist, probs_sort[:n_ref, 1], probs_sort[n_ref:, 1]
+
+    def _score_rf(self, x: np.ndarray) -> Tuple[float, float, np.ndarray, np.ndarray]:
+        x_ref, x = self.preprocess(x)
+        x, y, _ = self.get_splits(x_ref, x, return_splits=False)
+        self.model.fit(x, y)
+        # it is possible that some inputs do not have OOB scores. This is probably means that too few trees were
```

Comment:
If we are not going to raise an error, I think we should at least raise a warning?

Reply:
there is a warning raised by `sklearn` saying that some values are `NaN`.

Reply:
Ah yes sorry, the `"Some inputs do not have OOB scores. This probably means too few trees were used to compute any reliable OOB estimates"`. I think that's actually probably descriptive enough for us to just leave it at that?

Reply:
Agreed